src/object detection.py

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+get_ipython().system('pip install torch==1.8.1+cu111 torchvision==0.9.1+cu111 torchaudio===0.8.1 -f https://download.pytorch.org/whl/lts/1.8/torch_lts.html')
```

Comment:
I guess you're trying to convert the ```.ipynb``` to ```.py``` file, it's fine you can just put the ```.ipynb``` ( jupyter file) instead of converting it.

Reply:
@ShreyaMittalSM, resolve this conversation ASAP, so that i can merge your PR.